static/scss/cms/faculty.scss

```diff
@@ -0,0 +1,28 @@
+.faculty-block {
+  color: #505050;
```

Comment:
Can we please move to sass variables for color reuse?

Reply:
sure.